internal/service/networkpeering/resource_network_peering_test.go

```diff
@@ -64,6 +64,50 @@ func TestAccNetworkRSNetworkPeering_basicAzure(t *testing.T) {
 	})
 }
 
+func TestAccNetworkRSNetworkPeering_updateBasicAzure(t *testing.T) {
+	acc.SkipTestForCI(t) // needs Azure configuration
```

Comment:
:( is it hard to have this information in our CI setup so that we can actually have it running?

Reply:
Main issue I found while running them locally is the limitation of network_container that in AWS and Azure there can only be one container per cloud provider region and that you can't delete a network peering container if your project contains clusters. I'll work around that so that we can have the tests run in the CI

Reply:
in case it can help, we improved cleanup process to delete endpoints: https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/scripts/cleanup-test-env.sh#L19
we could improve it to delete these projects as well if needed

Reply:
Done! Now the tests run in the CI

Reply:
Thank you very much!